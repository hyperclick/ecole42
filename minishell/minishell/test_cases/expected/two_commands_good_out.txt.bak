/mnt/c/git/e57/minishell/minishell
echo3
echo5
echo6
echo7
echo8
