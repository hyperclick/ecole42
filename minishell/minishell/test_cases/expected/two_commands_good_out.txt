/mnt/c/git/e47/minishell/minishell
echo1
echo2
echo3
echo5
echo6
echo7
echo8
